README.rst

```diff
@@ -130,6 +130,8 @@ Detailed `installation instructions`_ are available.
 
 3. Install the `vagrant-service-manager`_ Vagrant plugin:
 
+   ``sudo dnf install libvirt-devel``
```

Comment:
But the documentation is for the VirtualBox provider here. There is no need to install/use libvirt if you are going to use VirtualBox, right?